Headers/AppKit/NSCell.h

```diff
@@ -123,6 +123,16 @@ enum {
 };
 typedef NSUInteger NSCellStateValue;
 
+/*
```

Comment:
Should we add a version guard around this?

Reply:
I believe so.  I will track down when this change was made and do that.

Reply:
Done